packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2849,6 +2849,24 @@ class RenderMouseRegion extends RenderProxyBox implements MouseTrackerAnnotation
     }
   }
 
+  @override
+  bool valid = true;
```

Comment:
Probably make it a getter that returns a mutable property `_validForMouseTracker`.